astropy/coordinates/builtin_frames/intermediate_rotation_transforms.py

```diff
@@ -85,24 +95,24 @@ def gcrs_precession_mat(equinox):
     return erfa.fw2m(gamb, phib, psib, epsa)
 
 
-def get_location_gcrs(reference, gcrs_to_reference_matrix):
-    """Create a GCRS frame at the location and obstime of a given reference.
+OMEGA_EARTH = u.Quantity(7.292115855306589e-5, 1./u.s)
```

Comment:
Actually, still defined in `earth.py`, although it is not used there. I wonder if we should revert to the old code there too...

Reply:
Yes - let’s leave earth.py alone for now.